src/monodroid/jni/monodroid-glue.cc

```diff
@@ -2268,16 +2249,16 @@ extern "C" void monodroid_dylib_mono_free (DylibMono *mono_imports)
 
   it should also accept libmono_path = nullptr parameter
 */
-extern "C" int monodroid_dylib_mono_init (DylibMono *mono_imports, const char *libmono_path)
+extern "C" int monodroid_dylib_mono_init (void *mono_imports, const char *libmono_path)
```

Comment:
Who/what is invoking this function that we need to keep it around?

Reply:
The only caller I found is in `Java.Interop`'s [`java-interop-gc-bridge-mono.cc`](https://github.com/xamarin/java.interop/blob/29f9707577753e577761b214b39294c1fda9fad7/src/java-interop/java-interop-gc-bridge-mono.cc#L272). It can probably be removed from that source, but there's no knowing if anyone else out there calls the function...